feature/experimental/gribi/otg_tests/encap_decap_scale/encap_decap_scale_test.go

```diff
@@ -561,27 +561,33 @@ func createAndSendTrafficFlows(t *testing.T, args *testArgs, decapEntries []stri
 		outHdrSrcIP: ipv4OuterSrc222, outHdrDstIP: decapStartIP.IP.String(), outHdrDscp: []uint32{dscpEncapD1},
 	})
 
-	flow5 := createFlow(&flowArgs{flowName: "flow5", isInnHdrV4: false, outHdrDstIPCount: decapRouteCount,
-		InnHdrSrcIPv6: atePort1.IPv6, InnHdrDstIPv6: encapVrfAIPv6Enries, inHdrDscp: []uint32{dscpEncapA2},
-		outHdrSrcIP: ipv4OuterSrc111, outHdrDstIP: decapStartIP.IP.String(), outHdrDscp: []uint32{dscpEncapA2},
-	})
+	// Below code will be un commented once ixia issue is fixed.
+	// https://github.com/open-traffic-generator/fp-testbed-juniper/issues/49
+	/*
+		flow5 := createFlow(&flowArgs{flowName: "flow5", isInnHdrV4: false, outHdrDstIPCount: decapRouteCount,
```

Comment:
@xw-g , I have un-commented code .  I think to access ixia issue you need to access from ixia for that repo.

Here is the  description I have added in ixia issue.

Hi,

I am sending ipv6 flow with traffic class 18, 28, 38 and 48 from OTG. But they are received on DUT as 4, 7,9 and 12.
is there any conversion happening internally inside otg? We want to receive the values as we set in script otherwise firewall
match wont happen.

Code used in script:

if len(flowValues.inHdrDscp) != 0 {
innerIpv6Hdr.TrafficClass().SetValues(flowValues.inHdrDscp)
}